dotCMS/src/main/java/com/dotmarketing/db/LocalTransaction.java

```diff
@@ -147,7 +148,34 @@ static public void wrap(final VoidDelegate delegate) throws Exception {
             }
         }
     } // wrap.
+    
+    static public void wrapNoException(final VoidDelegate delegate)  {
 
+        final boolean isNewConnection    = !DbConnectionFactory.connectionExists();
+        boolean isLocalTransaction = false;
+        
+        try {
+            isLocalTransaction =  DbConnectionFactory.startTransactionIfNeeded();
+            delegate.execute();
+            if (isLocalTransaction) {
+                DbConnectionFactory.commit();
+            }
+        } catch (Exception e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)